### PR TITLE
Add blackbox rules

### PR DESCRIPTION
**Pull request by mxssl**
#19

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -292,3 +292,11 @@ services:
       - name: prometheus/blackbox_exporter
         doc_url: https://github.com/prometheus/blackbox_exporter
         rules:
+          - name: Status Code
+            description: HTTP status code isn't equal to 200
+            query: 'probe_http_status_code != 200'
```
Comment (samber) [inline _data/rules.yml]: Can you check if status code is between 200 and 300 please ?

File: _data/rules.yml [hunk 2]
```diff
@@ -292,3 +292,11 @@ services:
...
+            severity: error
+          - name: SSL Cert expiration
```
Comment (samber) [inline _data/rules.yml]: Could you please duplicate this alert with `error` severity for expired certificates ?

File: _data/rules.yml (hunk 1 above)
Comment (mxssl) [reply _data/rules.yml]: done

File: _data/rules.yml (hunk 2 above)
Comment (mxssl) [reply _data/rules.yml]: done

**Comment by samber** [comment]
Merged!

Thanks ;)

**Pull request merged**